reanalysis/html_builder.py

```diff
@@ -359,21 +359,44 @@ class Variant:
     Try not to put presentation logic here - keep it in the jinja templates
     """
 
+    def get_var_change(self) -> str:
+        """
+        Find the variant change for the variant
+        - we want to truncate huge small variant InDels (ballooning column width)
+           - e.g. LOLOLOLOLOLOLOLOLOLOLOLOLOLOLOLOLO->A -> del 34bp
+        - SVs always need to be presented differently
+           - e.g INS 4079bp
+        """
+        if isinstance(self.var_data, SmallVariant):
+            if len(self.ref) > 10 or len(self.alt) > 10:
+                ref_len = len(self.ref)
+                alt_len = len(self.alt)
+                if ref_len > alt_len:
+                    return f'del {ref_len - alt_len}bp'
+                return f'ins {alt_len - ref_len}bp'
```

Comment:
If the ref and alt are the same length (and >10bp) will we see "ins 0bp" in this field? Is that fine?

Reply:
I'll give that a little tweak - I don't think I've ever seen a variant with both huge ref and alt alleles, but it's possible

Reply:
this scenario will result in `complex delins nbp`